modules/custom-post-types/portfolios.php

```diff
@@ -400,6 +401,7 @@ static function portfolio_shortcode( $atts ) {
 			'display_types'   => true,
 			'display_tags'    => true,
 			'display_content' => true,
+			'display_author' => true,
```

Comment:
We might want to default to `false`, as this will change the front-end appearance of many sites that already display portfolios.